mobilizon_reshare/aerich_conf/database.py

```diff
@@ -0,0 +1,24 @@
+"""tortoise orm conf for aerich"""
+from pathlib import Path
+from mobilizon_reshare.config.config import get_settings
+
+CONF_FILE = "mobilizon_reshare/settings.toml"
+
+SETTINGS = get_settings(CONF_FILE)
+db_path = Path(SETTINGS.db_path)
+print(db_path)
+
+db_url = f"sqlite:///{db_path}"
```

Comment:
tutta questa roba ti serve qui? non è replicata con l'altro file? se viene scomodo importare da lì, fai una funzione comune

Reply:
in attesa di dettagli da Giacomo su come sia possibile, runtime, leggere il parametro db_path nei settings e se è possibile farlo senza specificare path fisso del file di configurazione

Reply:
fatto